src/functions.php

```diff
@@ -17,12 +17,24 @@
     define('OpenApi\Annotations\UNDEFINED', UNDEFINED);
     define('OpenApi\Processors\UNDEFINED', UNDEFINED);
 
+    /**
+     * Determines whether the given string has openapi definitions or not
+     *
+     * @param string $filecontent File content to analyze
+     * @return boolean
+     */
+    function hasOpenApiDefinitions($filecontent)
+    {
+        return strpos($filecontent, '@OA\\') !== false;
```

Comment:
This will not work for the custom annotations under other namespaces

Reply:
true - i'll close this PR

Reply:
You can try get info from `Analyser::defaultImports` and `Analyser:whitelist` properties. But we should investigate this and coverage all cases with tests.